pkg/injector/webhook.go

```diff
@@ -180,20 +230,30 @@ func (wh *webhook) mutateHandler(w http.ResponseWriter, req *http.Request) {
 	log.Debug().Msg("Done responding to admission request")
 }
 
-func (wh *webhook) mutate(req *v1beta1.AdmissionRequest) *v1beta1.AdmissionResponse {
+func (wh *webhook) mutate(req *v1beta1.AdmissionRequest) (*v1beta1.AdmissionResponse, string) {
```

Comment:
what string is being returned here is unclear.

Reply:
that's probably what I like the least. I wanted something to identify the webhook to see what actual resource took more/less to execute. I'm basically returning the podname in this case.

If you some better idea or other thoughts let me know.

Reply:
We will not know the pod name at this time because the pod isn't even  created, so some arbitrary autogenerated name might not be useful.